### DISCUSSION TITLE
HmIP-BWTH Temperatur & Luftfeuchte in Homekit anzeigen

### DISCUSSION THREAD

**Discussion by wum-007** [thkl/hap-homematic#289] [Q&A] [answered]
Zuallererst möchte ich mich für das super Add-On für die Raspberrymatic bedanken. Ich bin neu im Forum und habe bisher keine Lösung für mein Problem gefunden. Ich habe den BWTH eingebunden. Bei den Services muss man ja entweder HomeMaticRadiatorThermostatAccessory oder HomeMaticHumidityAccessory auswählen. Gibt es eine Möglichkeit beide Services in Homekit einzubinden? Kann man ggf. ein Gerät virtualisieren und ein zweites Mal einer Instanz hinzufügen? Tausend Dank im Voraus!

**Comment by detLAN** [comment, marked as answer]
Beim Thema Heizen arbeitet Homematic mit Gruppen (HmIP-Heizungssteuerung). Dort werden alle relevanten Geräte (Heizkörperthermostat, Wandthermostat und Kontaktsensor) eines Raums zusammengefasst. Und diese Gruppe sollte dann auch zur Steuerung der Heizung genutzt werden. Wenn Du so eine Gruppe an via HAP zu HomeKit hinzufügst, hast Du auch alle relevanten Daten dort abrufbereit. Die Temperatur wird direkt und die Luftfeuchte unter Einstellungen des Geräts angezeigt.
Möchtest Du nun die Feuchte getrennt angezeigt bekommen, kannst Du zusätzlich das Wandthermostat via HAP mit dem entsprechenden Service zu HomeKit hinzufügen. Beachte aber, dass die Luftfeuchte in HomeKit ausschließlich oben unter Klima eingebunden wird.

**Reply by wum-007** [reply]
Vielen Dank für deine Erläuterung, Detlef. Sorry, wenn ich da nochmal nachhake - ich bin neu in dem Thema :-) Wie ich das Wandthermostat via HAP einbinde, habe ich herausgefunden und so habe ich diesen bisher diesen auch eingebunden und sehe die Temperatur und kann auch umstellen auf Anzeige der Luftfeuchte. Ich habe nun eine Gruppe für die Heizungssteuerung eines Raumes in RaspberryMatic angelegt und den Thermostat des Raumes der Gruppe hinzugefügt. Was ich nicht weiß, wie ich die Gruppe nun in HAP hinzufügen kann, um das Thermostat zusätzlich einzubinden, um die Luftfeuchte anzuzeigen. Kannst du mir da nochmal helfen?

**Reply by detLAN** [reply]
Du hast auf der RaspberryMatic also etwas wie dies hier eingerichtet:
![Heizungsgruppe0](https://user-images.githubusercontent.com/64675966/103142932-c2e68900-470c-11eb-91ea-a09e58c40e7b.jpg)

Dann musst Du unter Einstellungen / Geräte nach HEATING filtern (gelber Pfeil):
![Heizungsgruppe1](https://user-images.githubusercontent.com/64675966/103142901-55d2f380-470c-11eb-997a-1748d30f7ddf.jpg)

Auf der HAP-HomeMatic Weboberfläche unter Devices klickst Du auf die Schaltfläche "Neu" und suchst nach dem Namen der Heizungsgruppe (grüner Rahmen).

**Reply by wum-007** [reply]
Ich hatte es genauso gemacht, scheiterte aber an dem hinzufügen des Thermostats aus der Gruppe. HAP hat immer nur das Thermostat, das dem Raum als einzelnes Gerät zugeordnet war, aber nicht das Thermostat aus der Gruppe gefunden. Ich habe nun den Thermostat nochmal aus HAP entfernt und siehe da, beim Versuch diesen wieder hinzuzufügen, hat HAP mir auch die Geräte aus der Gruppe angezeigt. Tausend Dank!

**Comment by Matzushub** [comment]
Vielleicht kann ich die Frage noch mal aufmachen.
Wenn ich die Heizungsgruppe in HAP einpflege, welchen Kanal muss ich da nehmen? Ich nutze den Kanal Verkünpfungspartner Thermostat Sender. Bei mir wird zwar unter Einstellungen eine rel Feuchte angezeigt aber die steht immer nur konstant auf 20%. @detLAN 

meine Heizungsgruppe besteht aus Heizungsthermostat HM-CC-RT-DN
und Wandthermostat HM-TC-IT-WM-W-EU
